plugins/org.jkiss.dbeaver.ui.app.winstore/src/org/jkiss/dbeaver/winstore/WinStoreEnvPathMapper.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2022 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.winstore;
+
+import java.nio.file.Path;
+
+import org.jkiss.code.NotNull;
+import org.jkiss.dbeaver.Log;
+import org.jkiss.dbeaver.runtime.IEnvironmentPathMapper;
+import org.jkiss.dbeaver.utils.RuntimeUtils;
+
+public class WinStoreEnvPathMapper implements IEnvironmentPathMapper {
+    private static final Log log = Log.getLog(WinStoreEnvPathMapper.class);
+
+    private static final String windowsAppLocalDataPackage = "DBeaverCorp.DBeaverCE_1b7tdvn0p0f9y";
```

Comment:
Are you sure that will not change after the new version? I am not sure that it is hardcoded.

Reply:
It's generated from the package and publisher identify information in the application package manifest.
As far as I know, it won't be changed while we change the information in the application package manifest.

Reply:
Looks like a thing we will forgot about when we will try to change the package identity :(